aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +31,93 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
```

Comment:
Icons can be used to represent information to users. such as actions a user can perform and represent different types of functionality. It is important that the use of an icon is clear. In some cases, it may be beneficial to pair icons with text to ensure the use of the icon is properly understood.